### PR TITLE
Remove duplication

### PR DESCRIPTION
**Pull request by ChristianPauly**
# Description of the Change
This error causes that you are no longer able to compile this app. So I have removed all duplications. 

# Benefits
Now its possible again to run the app with clickable.

# Possible Drawbacks
Maybe the chinese translation needs now a new review.

Related Issues: #38

### CODE REVIEW

**Comment by bhdouglass** [comment]
@Flohack74 does something need to be fixed on the weblate side of things for this?

**Comment by Flohack74** [comment]
No it will be all good since no ids being renamed. You can always add and remove translation units, but renaming is snot so easy, you will probably loose the translation then.

File: po/zh_CN.po [hunk 1]
```diff
@@ -32,18 +32,28 @@ msgid "Version: "
 msgstr "版本： "
 
 #: ../qml/About.qml:68
+#, fuzzy
+#| msgid ""
```
Comment (Flohack74) [inline po/zh_CN.po]: Please delete commented lines.

File: po/zh_CN.po [hunk 2]
```diff
@@ -212,17 +142,24 @@ msgid ""
...
+"重新控制你的设备和个人资料。您有选择和自由使用 UBports。"
 
 #: ../qml/Home.qml:67
 msgid "Be more than a spectator - Become a game changer!"
 msgstr "不仅仅是一名观众，更是成为一名游戏的改变者！"
 
 #. TRANSLATORS: Please make sure the URLs are correct and don't fuck up the HTML.
 #: ../qml/Home.qml:73
+#, fuzzy
+#| msgid ""
```
Comment (Flohack74) [inline po/zh_CN.po]: Please delete commented lines.

**Comment by bhdouglass** [comment]
@ChristianPauly can you make the requested changes and resolve the conflicts? Then we should be good to merge this in.

**Comment by bhdouglass** [comment]
@ChristianPauly are you able to resolve these conflicts? I'm looking to get this merged so we can move this app to gitlab.

**Pull request closed without merging**